shared/src/main/scala-2.x/src/main/scala/mouse/all.scala

```diff
@@ -3,16 +3,17 @@ package mouse
 trait AllSharedSyntax
     extends AnySyntax
     with AnyFSyntax
-    with OptionSyntax
     with BooleanSyntax
-    with StringSyntax
-    with TrySyntax
-    with IntSyntax
-    with LongSyntax
     with DoubleSyntax
-    with PartialFunctionLift
-    with MapSyntax
-    with FOptionSyntax
+    with FBooleanSyntax
     with FEitherSyntax
     with FNestedSyntax
+    with FOptionSyntax
+    with IntSyntax
+    with LongSyntax
+    with MapSyntax
+    with OptionSyntax
+    with PartialFunctionLift
+    with StringSyntax
+    with TrySyntax
```

Comment:
I sorted them in alphabetical order, I hope that was a good idea... I don't know too much about binary compatibility.